src/rofi_rbw/typer/typer.py

```diff
@@ -33,7 +33,7 @@ def get_active_window(self) -> str:
         pass
 
     @abstractmethod
-    def type_characters(self, characters: str, key_delay: int, active_window: str) -> None:
+    def type_characters(self, characters: str, key_delay: int, type_delay: float, active_window: str) -> None:
```

Comment:
1. The order should be `characters`, `type_delay`, `key_delay`, `active_window` in my opinion. That makes more sense than the other way round.
2. Here, too, I'd like to call it `start_delay`.